README.md

```diff
@@ -16,12 +14,54 @@ Event data sets typically associate events with a certain subject (i.e. a person
 
 Conquery supplies a powerful interface to group event types in a hierarchical *concept tree* structure. Elements of this tree represent a group of similar subjects. Those subjects can be composed into a powerful query that runs against the data set.
 
+## Requirements
+- Maven 3 (optional for building)
+- Java JDK 11
+- Node.js 16 + Yarn
+- curl (to import the test data)
+
 ## Starting the demo
 
+### Frontend only
+
 This repository includes the Conquery frontend along with a non-functional backend. It provides a set of example concept trees to demonstrate the capabilities of the UI: The example's use case is to search for groups of actors who appeared in movies of the same genre or received the same award.
 
 Check the README in `/frontend` for details.
 
+### Frontend + Backend
+
+#### Steps
+To test frontend and backend together you can start the setup that is used for end-to-end tests.
+
+First build the backend using `conquery/scripts/build_version.sh` or download a JAR from the [release page](https://github.com/bakdata/conquery/releases) and place it in `conquery/executable/target/`.
```

Comment:
```suggestion
First build the backend using `./scripts/build_version.sh` or download a JAR from the [release page](https://github.com/bakdata/conquery/releases) and place it in `./executable/target/`.
```

Reply:
I thought about that, but I decided to be explicit because there are multiple `scripts` folder